modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -0,0 +1,1598 @@
+
+
+.Create a composite relationship property uniqueness constraint on several properties
+======
+
+.Create a constraint requiring `PREQUEL_OF` relationships to have unique `order` and `author` properties
+[source, cypher]
+----
+CREATE CONSTRAINT prequels
+FOR ()-[prequel:PREQUEL_OF]-() REQUIRE (prequel.order, prequel.author) IS UNIQUE
+----
+
+.Result
+[source, queryresult]
+----
+Added 1 constraint.
+----
+
+======
+
+[[create-property-uniqueness-constraint-compliant-data]]
+==== Create data that complies with existing property uniqueness constraints
+
+.Create a node that complies with existing property uniqueness constraints
+======
+
+.Create a `Book` node with an `isbn` that is not already in the graph
+[source, cypher]
+----
+CREATE (book:Book {isbn: '1449356265', title: 'Graph Databases'})
+----
+
+.Result
+[source, queryresult]
+----
+Added 1 label, created 1 node, set 2 properties
+----
+
+======
+
+
+.Create a relationship that complies with existing property uniqueness constraints
+======
+
+.Create a `SEQUEL_OF` relationship with an `order` property not already in the graph
+[source, cypher]
+----
+CREATE (:Book {title: 'Spirit Walker'})-[:SEQUEL_OF {order: 1, seriesTitle: 'Chronicles of Ancient Darkness'}]->(:Book {title: 'Wolf Brother'})
+----
+
+.Result
+[source, queryresult]
+----
+Added 2 labels, created 2 nodes, set 4 properties, created 1 relationship.
+----
+
+======
+
+[role=label--enterprise-edition]
+[[create-property-existence-constraints]]
+=== Create property existence constraints
+
+Property existence constraints ensure that a property exists either for all nodes with a specific label or for all relationships with a specific type.
+Queries that try to create new nodes of the specified label, or relationships of the specified type, without the constrained property will fail.
+The same is true for queries that try to remove the mandatory property.
+
+* xref:constraints/managing-constraints.adoc#create-single-property-existence-constraint[]
+* xref:constraints/managing-constraints.adoc#create-property-existence-constraint-compliant-data[]
+
+
+[[create-single-property-existence-constraint]]
+==== Create a single property existence constraint
+
+Property existence constraints on single properties are created with the following commands:
+
+* Node property existence constraint: `CREATE CONSTRAINT constraint_name FOR (n:Label) REQUIRE (n.property) IS NOT NULL`.
+* Relationship property existence constraint: `CREATE CONSTRAINT constraint_name
+FOR ()-[r:REL_TYPE]-() REQUIRE r.property IS NOT NULL`.
+
+For the full command syntax to create an existence constraint, see xref:constraints/syntax.adoc#create-property-existence-constraints[Syntax -> Create property existence constraints].
+
+[NOTE]
+It is not possible to create composite existence constraints on several properties.
+
+.Create a node property existence constraint
+======
+
+A node property existence constraint ensures that certain nodes have a specified property.
```

Comment:
both of these examples have this kind of sentence, (most of) the property uniqueness examples doesn't, do we want to be consistent around this?